src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -231,12 +253,15 @@ navigator.bluetooth.requestDevice({ filters: [{ services: ['heart_rate'] }] })
   var resetEnergyExpended = new Uint8Array([1]);
   return characteristic.writeValue(resetEnergyExpended);
 })
-.then(() => {
+.then(_ => {
   console.log('Energy expended has been reset.');
 })
 .catch(error => { console.log(error); });
 {% endhighlight %}
 
+Using a `characteristicvaluechanged` event listener to know when a characteristic
```

Comment:
Is https://bugs.chromium.org/p/chromium/issues/detail?id=592659 wrong then?
I'll also update our web Bluetooth sample as well then.